pyaedt/generic/configurations.py

```diff
@@ -1469,7 +1469,7 @@ def _export_mesh_operations(self, dict_out):
         pass
 
     @pyaedt_function_handler()
-    def _update_monitor(self, m_case, m_object, m_quantity, m_name):
+    def update_monitor(self, m_case, m_object, m_quantity, m_name):
```

Comment:
if you move from private to public a method you should add a docstring to describe it.